### PR TITLE
[easy] re-export crates being used in Kimchi

### PR DESCRIPTION
**Pull request by querolita**
This PR re-exports the crates being used in kimchi so that an external user does not need to write all the dependencies inside their `Cargo.toml`, but instead, only use 

```rust
kimchi = { git = "https://github.com/o1-labs/proof-systems" }
```
and similarly for kombucha
```rust
circuit-construction = { git = "https://github.com/o1-labs/proof-systems" }
```

Partially addresses [this issue](https://github.com/o1-labs/proof-systems/issues/527). The examples for the README.md will be moved to the readme in the circuit api instead.

### CODE REVIEW

File: kimchi/README.md [hunk 1]
```diff
@@ -53,3 +53,13 @@ To obtain a flamegraph:
     the [binary](src/bin/flamegraph.rs) will run forever, so you have to C-c to exit and produce the `flamegraph.svg` file.
 
 Note: lots of good advice on system performance in the [flamegraph repo](https://github.com/flamegraph-rs/flamegraph#systems-performance-work-guided-by-flamegraphs).
+
+
+## Guidelines
+
+When using this library, make sure to include in your Cargo.toml the following dependency:
+
+`
```
Comment (mimoo) [inline kimchi/README.md:62]: you need triple backtick here, followed with `toml` for syntax highlighting

File: kimchi/README.md [hunk 2]
```diff
@@ -53,3 +53,13 @@ To obtain a flamegraph:
     the [binary](src/bin/flamegraph.rs) will run forever, so you have to C-c to exit and produce the `flamegraph.svg` file.
 
 Note: lots of good advice on system performance in the [flamegraph repo](https://github.com/flamegraph-rs/flamegraph#systems-performance-work-guided-by-flamegraphs).
+
+
+## Guidelines
+
+When using this library, make sure to include in your Cargo.toml the following dependency:
```
Comment (mimoo) [inline kimchi/README.md:60]: It'd be nice to have this in the main `README.md` before the example snippet :o

File: kimchi/src/lib.rs [hunk 3]
```diff
@@ -1,8 +1,15 @@
-#![doc = include_str!("../README.md")]
+#![doc = include_str!("../../README.md")]
```
[the remark this answers is not in the document]
Comment (querolita) [reply kimchi/src/lib.rs:1]: Oh, no idea... Not sure where this came from

**Review by mimoo — APPROVED** [review]
LGTM!

**Pull request merged**